ui/src/app/components/apps-panel/expanded-app.tsx

```diff
@@ -134,14 +150,6 @@ const CromwellButtonRow = (props: {
   );
 };
 
-// TODO: refine and style more like RuntimeStatusIcon
```

Comment:
we still want this eventually, but for now I moved it inline with the "refresh" text